mozilla/addons-server
# Store a copy of the manifest in the database for every File we create

Fixes: mozilla/addons#14919

### Testing

- Upload a new add-on and/or version: you should be able to see a `FileManifest` entry for the corresponding `File` in the admin (`/en-US/admin/models/files/file/`)
- Make sure you have some real add-ons in your local environment, with xpi/zip present in storage (you can also have some fake ones with no files - the backfill command will ignore those). If you didn't have any before testing this PR, creating new versions would automatically create the `FileManifest`, so use the django shell to delete at least one to make sure the backfill task will have something to do.
- Run the `manage.py process_files --task=backfill_file_manifest` command, check `docker-celery.log` logs to see if it succeeded. Then check the admin as described above, you should have one `FileManifest` for every `File` that has a xpi on storage in your environment (if you happen to have a lot of files it may take a while)
